docs/knowledge-base/posts/routing-l2-networks.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Routing Layer 2 Networks with VergeOS
+slug: routing-l2-networks
+description: Describes an overview and specific scenario in which physical devices are routed via Layer 3 Verge networks.
+draft: false
+date: 2025-03-21T18:42:31.354Z
+tags:
+  - network
+  - networking
+  - network blocks
+  - subnet
+  - virtual wire
+  - vwire
+  - layer2
+  - layer 2
+  - L2
+  - l2
+categories:
+  - Network
+editor: markdown
+dateCreated: 2025-03-21T11:15:31.354Z
+---
+## Routing IP Traffic to L2 physical networks
+This article will walk through an efficient method of routing layer 3 IP traffic for layer 2 physical networks with the powerful VergeOS networking system. There are multiple ways to achieve this, however the objective of this article is to provide clear and concise guidance on a specific common scenario. This article should be particularly of use and interest to engineers and administrators of Edge deployments in  in which physical devices like phones, cctv cameras, or network equipment require communication with workloads in VergeOS that may be running on different IP subnets. Using this method, an operator can achieve direct local communication between those physical devices outside of VergeOS and the internal VergeOS managed networks.
+    
+### Steps Overview
+1. [Create Layer2 external Network](/knowledge-base/network-blocks/)
+	- It is important that this network is created with no IP information. If the IP block is already assigned to this interface, it cannot be added to the default route in the next step.
+2. [Create IP block on the external network](/knowledge-base/network-blocks/?h=block#creating-a-network-block) that will route the traffic (Likely default External)
+	- By adding the IP block here, we're telling VergeOS to expect this IP traffic to come in via this interface, and in our next step we will tell Verge networking where to send it.
+	- Assign IP block to step 1 external network (or tenant)
+	- Upon saving, VergeOS will automatically create route(s) to send inbound traffic with matching destination IPs to our Layer2 network which may also have VMs attached, allowing all clients to communicate across networks within Verge (with appropriate firewall rules) and to networks outside of Verge via the default route.
```

Comment:
It’s technically NAT-less bridging, not routing in the traditional L3 sense. You might want to mention that